O21.Game/Localization/Translations.fs

```diff
@@ -0,0 +1,16 @@
+module O21.Localization.Translations
+
+open FSharp.Data
+
+type Language = 
+    | English
+    | Russian
+
+type private Provider = JsonProvider<"Localization\Translations\english.json">
+
+
+let Translation(language: Language) =
+    let rootFolder = "Localization\\Translations\\"
```

Comment:
This should be converted to a portable path as well: either use `/` or `Path.Combine`.

Also, I'd recommend to choose the root folder more carefully: currently, it seems to get resolved relatively to the current working directory, while it should be resolved relative to the game binary.